### PR TITLE
[RM Ch03] Update chapter03.md

### PR DESCRIPTION
**Pull request by walterkozlowski**

### CODE REVIEW

**Comment by walterkozlowski** [comment]
Removing Sec 3.2.4 Network, which is now replaced by 3.5 Network (see PR#1535)
The reason for this is that in 3.5 Network we talk about both virtual and physical  resources for networking (removed section 3.2.4 was under 3.2 Virtual Resources).

File: doc/ref_model/chapters/chapter03.md [hunk 1]
```diff
@@ -7,9 +7,8 @@
 * [3.2 Virtual Resources](#3.2)
   * [3.2.1 Tenant](#3.2.1)
   * [3.2.2 Compute](#3.2.2)
-  * [3.2.3 Storage](#3.2.3)
-  * [3.2.4 Network](#3.2.4)
-  * [3.2.5 Availability Zone](#3.2.5)
+  * [3.2.3 Storage](#3.2.4)
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter03.md:10]: "[3.2.3 Storage](#3.2.4)" should be "[3.2.3 Storage](#3.2.3 )"

"[3.2.4 Availability Zone](#3.2.5)" should be "[3.2.4 Availability Zone](#3.2.4  )"  (the extra space as Github keeps changing to an Issue -- but please delete)

Comment (rabi-abdel) [reply doc/ref_model/chapters/chapter03.md:10]: @pgoyal01 resolved in https://github.com/cntt-n/CNTT/pull/1536/commits/e3e5ca93fc3a528f62a84d60de15d024af6373c9

Comment (pgoyal01) [reply doc/ref_model/chapters/chapter03.md:10]: @rabi-abdel Thanks. 3.2.3 Storage still references 3.2.4 instead of 3.2.3

Comment (kedmison) [inline doc/ref_model/chapters/chapter03.md:10]:
```suggestion
  * [3.2.3 Storage](#3.2.3)
```

Comment (kedmison) [reply doc/ref_model/chapters/chapter03.md]: I committed a change to the anchor.

**Pull request merged**